tests/test_models.py

```diff
@@ -44,19 +44,39 @@ def _create_supplier(self):
             address="123 Main Street, Anytown USA", 
             email="jjones@gmail.com", 
             phone_number="800-555-1212",
-            product_list=[1,2,3,4]
+            products=[]
         )
 
+    def _create_supplier_with_product(self): 
+        supplier = Supplier(
+            name="Jim Jones",
+            address="123 Main Street, Anytown USA", 
+            email="jjones@gmail.com", 
+            phone_number="800-555-1212",
+            products=[]
+        )
+        association1 = Association(supplier_id = supplier.id, wholesale_price=999)
+        association.product = Product(name="Macbook")
+        association2 = Association(supplier_id = supplier.id, wholesale_price=999)
```

Comment:
why are these associations with the same values?

Reply:
Cleaned this up